languages/tree-sitter-stack-graphs-java/README.md

```diff
@@ -0,0 +1,27 @@
+# tree-sitter-stack-graphs definition for Java
+
+This project defines tree-sitter-stack-graphs rules for Java using the [tree-sitter-java](https://www.npmjs.com/package/tree-sitter-java) grammar.
+
+## Development
+
+The project is organized as follows:
+
+- The stack graph rules are defined in `src/stack-graphs.tsg`.
+- Tests are put into the `tests` directory.
+
+Make sure all development dependencies are installed by running:
+    ./bootstrap
+
+Run all tests in the project by executing the following:
+
+    cargo test
+
+Parse and test a single file by executing the following commands:
+
+    <!-- TODO: allow this option via cargo -->
+
+Additional flags can be passed to these commands as well. For example, to generate a visualization for the test, execute:
+
+    ./run -V
```

Comment:
Perhaps a separate `run` script is not necessary anymore. But if you want to keep it, I suggest renaming it to `script/test`. But it doesn't do much more than invoking `cargo run`, so I don't know if it helps or just confuses to have an extra script.